File: softuni-js-basics/02-conditional-statements/exercises-submitted-evaluated/07.areaOfFigures.js
function areaOfFigures(input) {
    let figure = input[0];
    let area = 0;
    let a = Number(input[1]);

    if (figure == "square") {
        area = Math.pow(a, 2);
    } else if (figure == "rectangle") {
        let b = Number(input[2]);
        area = a * b;
    } else if (figure == "circle") {
        area = (Math.pow(a, 2) * Math.PI);
    } else if (figure == "triangle") {
        let h = Number(input[2]);
        area = (a * h) / 2;
    }
    console.log(area.toFixed(3));
}

areaOfFigures(["square", "5"]);
areaOfFigures(["rectangle", "7", "2.5"]);
areaOfFigures(["circle", "6"]);
areaOfFigures(["triangle", "4.5", "20"]);

File: softuni-python-basics/02_conditional_statements/07_area_of_figures.py
from math import pi

figure = input()
dimension_1 = float(input())
result = 0

if figure == "square":
    result = dimension_1 ** 2
elif figure == "rectangle":
    dimension_2 = float(input())
    result = dimension_1 * dimension_2
elif figure == "circle":
    result = (dimension_1 ** 2) * pi
elif figure == "triangle":
    dimension_2 = float(input())
    result = dimension_1 * dimension_2 / 2

print(f"{result:.3f}")
